docs/source/installation/for_dev.rst

```diff
@@ -21,7 +21,7 @@ If you are a Windows developer and want to use PowerShell, you can use the below
 	
 	  (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python
 
-.. note:: Poetry can be installed by other ways also, such as with ``pip``. See `<https://python-poetry.org/docs/#alternative-installation-methods-not-recommended>`_.
+.. note:: Poetry can be installed by other ways also, such as with ``pip``. See `<https://python-poetry.org/docs/#alternative-installation-methods-not-recommended>`_. If you are using MacOS with the Homebrew package manager, you can also install poetry with ``brew install poetry``.
```

Comment:
I would rather link an issue here. https://github.com/Homebrew/homebrew-core/issues/53169

Reply:
I don't think that issue's relevant here. Quoting from it:

> by moving to installing Poetry with brew we're no longer able to point Poetry at Python versions managed by pyenv.

We aren't using PyEnv to manage our Virtual Environments, nor are we recommending other Manim developers do so. (At least, I don't _think_ we are. Has something changed in the past two months?)

Thoughts?

Reply:
Hmm. The issue says you can use things used with brew only. Say you install python 3.9 with brew and also peotry, then poetry will always find python 3.9 and can't use python installed using other means.

Reply:
Not exactly. Installing poetry via brew will _always_ install poetry with the latest version of python. See:

```shell
aathishs@Aathishs-MacBook-Air ~ % brew info poetry
poetry: stable 1.1.4 (bottled)
Python package management tool
https://python-poetry.org/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/poetry.rb
License: MIT
==> Dependencies
Required: python@3.9 ✘ 
==> Analytics
install: 5,005 (30 days), 7,755 (90 days), 16,534 (365 days)
install-on-request: 4,996 (30 days), 7,745 (90 days), 16,509 (365 days)
build-error: 0 (30 days)
aathishs@Aathishs-MacBook-Air ~ % 
```
Notice how Python3.9 is in the dependencies?